src/Console/Command/CommandHelp.php

```diff
@@ -312,6 +319,39 @@ public static function getDisplayableAllowedValues(FixerOptionInterface $option)
         return $allowed;
     }
 
+    public static function getLatestReleaseVersionFromChangeLog()
+    {
+        static $version = null;
+
+        if (null !== $version) {
+            return $version;
+        }
+
+        $currentMajor = (int) Application::VERSION;
+        $changelogFile = __DIR__.'/../../../CHANGELOG.md';
+
+        if (!is_file($changelogFile)) {
+            return null;
```

Comment:
no, as `getVersion` may provide you dev version that is not yet tagged, so all links with that version would not work.
all versions in changelog are tagged.